website_and_docs/content/project/governance/_index.html

```diff
@@ -60,30 +83,30 @@ <h3 id="users">Users</h3>
       <h3 id="contributors">Contributors</h3>
 
       Contributors are community members who contribute in concrete ways to the project,
-      most often in the form of code and/or documentation. Anyone can become a Contributor,
-      and contributions can take many forms, e.g.:
+      most often in the form of code and/or documentation in the respective project repository. 
```

Comment:
I don't know if this level of detail is necessary.

Reply:
ok